src/System.Net.Security/ref/project.json

```diff
@@ -9,9 +9,9 @@
     "System.Threading.Tasks": "4.0.0"
   },
   "frameworks": {
-    "netstandard1.3": {
+    "netstandard1.6": {
```

Comment:
Currently all the netstandard17 work is happening in the dev/api branch and is only a placeholder for now. We are going to soon be shifting to exposing APIs in .NET Core specifically first and then later adding them to netstandard. This enables people to add APIs just for .NET Core much more agile and not be blocked by our restrictions on .NET Standard.